bundles/net.tourbook.device/src/net/tourbook/device/sporttracks/FitLogSAXHandler.java

```diff
@@ -448,13 +544,84 @@ private void finalizeTour() {
          });
 
          tourData.setTourDescription(tourNotes.toString());
+         if (_currentActivity.customDataFields.containsKey("NormalizedPower [W]")) { //$NON-NLS-1$
+            try {
+               final float val = Float.parseFloat(_currentActivity.customDataFields.get("NormalizedPower [W]")); //$NON-NLS-1$
+               tourData.setPower_Normalized((int) val);
+               _currentActivity.powerNormalized = val;
+            } catch (final Exception exc) {
+               StatusUtil.log("NormalizedPower [W]" + ": FitlogEx parse error", exc); //$NON-NLS-1$ //$NON-NLS-2$
+            }
+         }
+         if (_currentActivity.customDataFields.containsKey("TSS (Training Stress Score)")) { //$NON-NLS-1$
+            try {
+               final float val = Float.parseFloat(_currentActivity.customDataFields.get("TSS (Training Stress Score)")); //$NON-NLS-1$
+               tourData.setPower_TrainingStressScore(val);
+               _currentActivity.powerTSS = val;
+            } catch (final Exception exc) {
+               StatusUtil.log("TSS (Training Stress Score)" + ": FitlogEx parse error", exc); //$NON-NLS-1$ //$NON-NLS-2$
+            }
+         }
+         if (_currentActivity.customDataFields.containsKey("IF (Intensity Factor)")) { //$NON-NLS-1$
+            try {
+               final float val = Float.parseFloat(_currentActivity.customDataFields.get("IF (Intensity Factor)")); //$NON-NLS-1$
+               tourData.setPower_IntensityFactor(val);
+               _currentActivity.powerIntensityFactor = val;
+            } catch (final Exception exc) {
+               StatusUtil.log("IF (Intensity Factor)" + ": FitlogEx parse error", exc); //$NON-NLS-1$ //$NON-NLS-2$
+            }
+         }
+         if (_currentActivity.customDataFields.containsKey("Left Torque Effectiveness Avg. [%]")) { //$NON-NLS-1$
```

Comment:
Per this [rule](https://rules.sonarsource.com/java/RSPEC-1192), I would recommend to define a constant instead of duplicating this literal.

Reply:
The same applies for a lot of the fields below